### DISCUSSION TITLE
`dfx identity info`

### DISCUSSION THREAD

**Discussion by hansl** [dfinity/sdk#2653] [Feature Requests]
Would show more information about the identity, such as:

```
name: default
principal: abcde-aa
path: /Users/hansl/.config/dfx/identity/default/identity.pem
```

Could also have a `--json` file for making this easier to script.

**Comment by gobengo** [comment]
'info' isn't a verb, and it's not a very useful noun. `dfx identity` or `dfx identity show` could be more in line with the 'noun-verb' pattern we talked about a few arch meetings ago.

wrt `--json`, I think we already use that somewhere so it's fine sugar, but I wonder if `--output json` is more scalable, e.g. to `--output yaml`. This is what kubectl does and works well, e.g. `kubectl get service foo -o json | jq .`. They also have a way of doing jq-like things with `-o jsonpath` without needing to bring-your-own jq. https://kubernetes.io/docs/reference/kubectl/cheatsheet/#formatting-output
